addons/core/translations/resources/en-us.yaml

```diff
@@ -111,6 +111,9 @@ host-set:
     filter: 
       label: Filter
       help: Create a filter to select resources using values such as tag name or tag value. Filter format is specific to each provider
+    preferred_endpoints:
```

Comment:
Can we add a `placholder` as well and make it a realistic example of an cidr or dns address?  Then add it to the field.

Reply:
sure. would something like `43.33.85.22` work?

Reply:
added!